content/case_histories/noranda/survey.rst

```diff
@@ -0,0 +1,8 @@
+.. _noranda_survey:
+
+Survey
+======
+
+"The entire 12.7 million cell model was forward modeled at 30, 45, 90, 180, 360, and 720 Hz."
```

Comment:
Maybe remember what type of survey is it and its characteristics with links to the appropriate geophysical surveys and fundamentals physics page. (ZTEM, Natural Sources, Airborne...)

Also, do we know how dense are the forward data? (one ZTEM logging at each cell...?)